pkgs/development/libraries/pystring/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation rec {
+  pname = "pystring";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "imageworks";
+    repo = "pystring";
+    rev = "v${version}";
+    sha256 = "1w31pjiyshqgk6zd6m3ab3xfgb0ribi77r6fwrry2aw8w1adjknf";
+  };
+
+  nativeBuildInputs = [ libtool ];
+
+  patches = [ ./makefile.patch ];
```

Comment:
Is this patch upstreamable?

Reply:
Dated 2020-11-20, where does it come from? Please use `fetchpatch` where possible

Reply:
> Dated 2020-11-20, where does it come from?

https://github.com/archlinux/svntogit-community/blob/packages/pystring/trunk/makefile.patch

+1 on `fetchpatch`ing that instead.

Reply:
Replaced this patch with a `fetchpatch`ed pull request on the upstream repo that adds a CMake build.